### PR TITLE
 Power Management WPEFrameWork Plugin build support added

### PR DESCRIPTION
**Pull request by Haseenamol**
WPEFrameWork Plugin build support added 
PMLIB selection updated based toolchain header flags

### CODE REVIEW

File: package/wpe/wpeframework-plugins/wpeframework-plugins.mk [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-WPEFRAMEWORK_PLUGINS_VERSION = 54bec7aff70825500c49ece273f871e4028b8d7f 
+WPEFRAMEWORK_PLUGINS_VERSION = 2039b2c6cfcecbb159f5289ba923c8a67dd9347b
```
Comment (wouterlucas) [inline package/wpe/wpeframework-plugins/wpeframework-plugins.mk]: Please drop the bumps, I think they point to another branch?

File: package/wpe/wpeframework/wpeframework.mk [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-WPEFRAMEWORK_VERSION = e3f46e7eede8457e11c9af5215f25914e42b95e2 
+WPEFRAMEWORK_VERSION = 8c01c446e09ba6652783c0f2f40cd2aba5363089
```
Comment (wouterlucas) [inline package/wpe/wpeframework/wpeframework.mk]: Same here, no need to bump unless it is behind. Seems to point at a hash on a branch?

File: board/bcm/bcm7252_full_defconfig [hunk 3]
```diff
@@ -205,6 +205,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_OHCI_HCD=y
 CONFIG_USB_STORAGE=y
+CONFIG_USB_USBNET=y
```
Comment (wouterlucas) [inline board/bcm/bcm7252_full_defconfig]: Is this needed for the Nexus power management lib?

Comment (Haseenamol) [reply board/bcm/bcm7252_full_defconfig]: It is to support Ethernet over USB

File: package/wpe/wpeframework-plugins/wpeframework-plugins.mk (hunk 1 above)
Comment (Haseenamol) [reply package/wpe/wpeframework-plugins/wpeframework-plugins.mk]: I will change this into hash of master branch

File: package/wpe/wpeframework/wpeframework.mk (hunk 2 above)
Comment (Haseenamol) [reply package/wpe/wpeframework/wpeframework.mk]: I will change this into hash of master branch

**Pull request merged**